app/experimenter/experiments/tests/api/v6/test_serializers.py

```diff
@@ -395,3 +396,31 @@ def test_serializer_outputs_empty_targeting(self):
         serializer = NimbusExperimentSerializer(experiment)
         self.assertEqual(serializer.data["targeting"], "true")
         check_schema("experiments/NimbusExperiment", serializer.data)
+
+    def test_serializer_rollouts_true(self):
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.LAUNCH_APPROVE,
+            publish_status=NimbusExperiment.PublishStatus.APPROVED,
+            targeting_config_slug=NimbusExperiment.TargetingConfig.NO_TARGETING,
+            application=NimbusExperiment.Application.FENIX,
+            firefox_min_version=NimbusExperiment.Version.FIREFOX_94,
+            is_rollout=True,
+        )
+
+        serializer = NimbusExperimentSerializer(experiment)
+        self.assertEqual(serializer.data["isRollout"], True)
+        check_schema("experiments/NimbusExperiment", serializer.data)
+
+    def test_serializer_rollouts_false(self):
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.LAUNCH_APPROVE,
+            publish_status=NimbusExperiment.PublishStatus.APPROVED,
+            targeting_config_slug=NimbusExperiment.TargetingConfig.NO_TARGETING,
+            application=NimbusExperiment.Application.FENIX,
+            firefox_min_version=NimbusExperiment.Version.FIREFOX_94,
+            is_rollout=False,
+        )
+
+        serializer = NimbusExperimentSerializer(experiment)
+        self.assertEqual(serializer.data["isRollout"], False)
+        check_schema("experiments/NimbusExperiment", serializer.data)
```

Comment:
I don't think you need these tests, the field is using entirely built in serializer functionality so just updating the above tests that the field is included should be sufficient.